deploy/olm-catalog/ocs-operator/manifests/storagecluster.crd.yaml

```diff
@@ -2386,12 +2326,6 @@ spec:
           properties:
             cephBlockPoolsCreated:
               type: boolean
-            cephFilesystemsCreated:
-              type: boolean
-            cephObjectStoreUsersCreated:
-              type: boolean
-            cephObjectStoresCreated:
-              type: boolean
```

Comment:
Except for these, all other changes in this file is wrong. 
Can you try generating again on top of latest master?